client/components/faq/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Internal dependencies
+ */
+
+import FAQ from './faq';
+import FAQItem from './faq-item';
+
+export default { FAQ, FAQItem };
```

Comment:
Isn't it better to have an `index` file in each component which summarizes what's in the component? Why do we import the files separately? Is the additional `index` file such an overhead?